docs/installation/mac.md

```diff
@@ -38,7 +38,7 @@ You can install ImageMagick with `brew install imagemagick`.
 
 ### Redis
 
-Forem requires Redis version 4.0 or higher.
+Forem requires Redis version 4.0 or higher to be running.
```

Comment:
```suggestion
Forem requires Redis version 4.x to run.
```

Reply:
Yeah, I like this suggestions! Thank you Molly.

Reply:
Updated with these suggestions along with the section on PostgreSQL that also had "to be running." Thanks for the suggestions @mstruve